src/test/java/com/amazon/janusgraph/triple/Triple.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2014-2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+package com.amazon.janusgraph.triple;
+
+import lombok.Getter;
+
+/**
+ *
+ * @author Addison Slabaugh
+ *
+ */
+public class Triple {
+
+    @Getter public static String[] line;
```

Comment:
I don't think the objects and properties in Triple can be final. The purpose of Triple is to take a single line from a TXT or CSV file and parse it into a triple of Left Object, Right Object and Relationship. Since we don't know those variables before the object is instantiated, we can't set them ahead of time -- so they shouldn't be final. Am I thinking about that correctly?